test/test_version_compare.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from cve_bin_tool.version_compare import Version
+
+
+class TestVersionCompare:
+    def test_eq(self):
+        """Make sure == works between versions"""
+        assert Version("1.2") == Version("1.2")
```

Comment:
Following up a comment I left somewhere else: what's the expected results of Version("1.1a")==Version("1.1A")?